ui/src/containers/DashboardCard/index.js

```diff
@@ -0,0 +1,244 @@
+import React, { Component } from 'react';
+import Scroll from 'react-scroll';
+import * as _ from 'lodash';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import { loggedInUserId } from 'ui/redux/selectors';
+import { actions, routeNodeSelector } from 'redux-router5';
+import { activeOrgIdSelector } from 'ui/redux/modules/router';
+import { COPY_DASHBOARD } from 'ui/redux/modules/dashboard/copyDashboard';
+import { withProps, compose, lifecycle, withHandlers, mapProps } from 'recompose';
+import { getVisualisationsFromDashboard } from 'ui/redux/modules/dashboard/selectors';
+import { BLANK_DASHBOARD, CURATR_STARTER } from 'lib/constants/dashboard';
```

Comment:
Sorry mate, this is the same issue as before with references to curatr and so the constants don't match. The other references in this file are on line 22 and line 142, and the filename of ui/src/containers/DashboardCard/assets/curatr-starter.png

```suggestion
import { BLANK_DASHBOARD, STREAM_STARTER } from 'lib/constants/dashboard';
```